tests/model_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for the Post model."""
+
+import pytest
+from freezegun import freeze_time
+from django.utils import timezone
+from blog.models import Post
+
+
+@pytest.fixture
+def post():
+    """Creates a test Post instance."""
+    return Post()
+
+
+@freeze_time('2023-01-01 12:00:00')
+def test_publish(post):
+    """Check publish() modifies published_date field."""
+    post.publish()
+    assert post.published_date is not None
+    assert post.published_date == timezone.now()
```

Comment:
Use constants in tests, don't add extra logic or computations. Tests should be as simple as possible. The more dynamic things you add, the more fragile they become.

Reply:
Marking as resolved.